src/modules/Visualizer/services/layersTreeUtils.js

```diff
@@ -4,18 +4,36 @@ export const INITIAL_FILTERS = new Map();
 
 export const isCluster = (source, layerId) => !!source.match(new RegExp(`^${layerId}-${PREFIX_SOURCE}-[0-9]+`));
 
-export function initLayersStateAction (layersTree) {
+/**
+ * Returns a flattened map of layers state from a layers tree
+ *
+ * @param {object} layersTree The layers tree config object
+ * @param {string|string[]} layers Active layers from hash (comma-separated)
+ * @param {string} table Active table from hash (layer id)
+ * @return {Map} A reduced layer tree state
+ */
+export function initLayersStateAction (layersTree, { layers, table } = {}) {
```

Comment:
Could you make it as an Arrow function instead?

```suggestion
export const initLayersStateAction = (layersTree, { layers, table } = {}) => {
```